docs/source/rdump.rst

```diff
@@ -0,0 +1,115 @@
+* limit the output
+* format the results
+* write the results through an adapter
+
+.. hint::
+
+    Don't know yet what a record is? Read more :ref:`here <overview/index:records>` for a detailed explanation.
+
+Field Selection
+~~~~~~~~~~~~~~~
+
+Imagine we're going to extract a list of users from a forensic image using:
+
+.. code-block:: console
+
+    $ target-query host.img -f users 
+
+To select only the name and home fields we use the ``-F`` option:
+
+.. code-block:: console
+
+    $ target-query host.img -f users | rdump -F name,home
+
+Using the ``-X`` option we can exclude fields:
+
+    $ target-query host.img -f users | rdump -X hostname,domain,sid
+
+
+Filtering
+~~~~~~~~~
+
+Filtering records can be done through the ``-s`` option.
+The selection option must be Python-expression, where the record is represented by the symbol ``r``.
+So to eliminate all records that have no domain value:
+
+.. code-block:: console
+
+    $ target-query host.img -f users | rdump -F name,home -s "r.domain is not None"
+
+
+Derivations
+~~~~~~~~~~~
+
+It is possible to create new fields derived from other fields. This can be done by using
+an expression and the ``-E`` option. The following example lists all dll files from
+a host and adds the filename to a separate field:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name"
+    
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgdk_pixbuf-2.0-0.dll' size=0.11 MB file='libgdk_pixbuf-2.0-0.dll'>
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgmodule-2.0-0.dll' size=26.6 KB file='libgmodule-2.0-0.dll'>
+    <filesystem/entry path='\sysvol\Program Files\Ethereal\libgobject-2.0-0.dll' size=0.25 MB file='libgobject-2.0-0.dll'>
+    ....and many more....
+
+
+Limiting
+~~~~~~~~
+
+The example above yields quite a lot records. Limiting the number of records can be done through
+the ``--count`` and ``--skip`` options:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name" --skip=1 --count=3
+
+    <filesystem/entry path='\sysvol\My Documents\COMMANDS\cygwinb19.dll' size=0.38 MB file='cygwinb19.dll'>
+    <filesystem/entry path='\sysvol\My Documents\ENUMERATION\NT\Cerberus\dnsscan.dll' size=40.0 KB file='dnsscan.dll'>
+    <filesystem/entry path='\sysvol\My Documents\ENUMERATION\NT\Cerberus\fingerscan.dll' size=40.0 KB file='fingerscan.dll'>
+
+Formatting
+~~~~~~~~~~
+
+A Python-style formatting rule can be specified using the ``-f`` option. The following example applies the format
+``{file} ({size})`` to the records:
+
+.. code-block:: console
+
+    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name" --skip=1 --count=3 -f "{file} ({size})"
+    
+    cygwinb19.dll (0.38 MB)
+    dnsscan.dll (40.0 KB)
+    fingerscan.dll (40.0 KB)
```

Comment:
```suggestion
    $ target-query host.img -f walkfs | rdump -s "r.path.suffix=='.dll'" -F path,file,size -E "file=path.name" --skip=1 --count=3 -f "Filename: {file} Size: {size}"
    
    Filename: cygwinb19.dll Size: 0.38 MB
    Filename: dnsscan.dll Size: 40.0 KB)
    Filename: fingerscan.dll Size: 40.0 KB
```
Would likely make the format and the customization possibilities more understandable.